src/components/Gradebook/gradebook.scss

```diff
@@ -41,24 +45,37 @@
 .gbook { 
     overflow-x: scroll;
 
+    .grade-button {
+        text-decoration: underline;
```

Comment:
UX/UI request: "For the grades themselves, suggest underlining the text. Affordance around this interaction will help with how intuitive it feels."